src/util/triplea/MapXMLCreator/MapPropertiesPanel.java

```diff
@@ -0,0 +1,360 @@
+package util.triplea.MapXMLCreator;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.FocusEvent;
+import java.awt.event.FocusListener;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.io.File;
+
+import javax.swing.AbstractAction;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import util.image.FileOpen;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public class MapPropertiesPanel {
+	
+	public static void layout(final MapXMLCreator mapXMLCreator, final JPanel m_stepActionPanel) {
+
+		if (!MapXMLHelper.s_xmlStrings.containsKey("info_@name"))
+		{
+			for (final String newKey : new String[] { "info_@name", "info_@version" })
+			{
+				MapXMLHelper.s_xmlStrings.put(newKey, "");
+			}
+		}
+		
+		if (MapXMLHelper.s_resourceList.isEmpty())
+			MapXMLHelper.s_resourceList.add("");
+
+		final JTextField tMapName = new JTextField(MapXMLHelper.s_xmlStrings.get("info_@name"));
+		final JTextField tMapVersion = new JTextField(MapXMLHelper.s_xmlStrings.get("info_@version"));
+		final JTextField tResourceName = new JTextField(MapXMLHelper.s_resourceList.get(0));
+		final JTextField tMapImageFile = new JTextField((MapXMLCreator.s_mapImageFile==null?"":MapXMLCreator.s_mapImageFile.getAbsolutePath()));
+		final JTextField tCentersFile = new JTextField((MapXMLCreator.s_mapCentersFile==null?"":MapXMLCreator.s_mapCentersFile.getAbsolutePath()));
+		final JTextField tWaterFilter = new JTextField(MapXMLCreator.s_waterFilterString);
+		final JLabel lMapName = new JLabel("Map Name:");
+		final JLabel lMapNameExample = new JLabel("e.g. 'Revised'");
+		final JLabel lMapVersion = new JLabel("Map Version:");
+		final JLabel lMapVersionExample = new JLabel("e.g. '1.2.0.1'");
+		final JLabel lResourceName = new JLabel("Resource Name:");
+		final JLabel lResourceNameExample = new JLabel("e.g. 'PUs'");
+		final JLabel lMapImageFile = new JLabel("Map Image File:");
+		final JButton bSelectMapImageFile = new JButton("Browse");
+		final JLabel lCentersFile = new JLabel("Map Centers File:");
+		final JButton bSelectCentersFile = new JButton("Browse");
+		final JLabel lWaterFilter = new JLabel("Water Territory Filter");
+		final JLabel lWaterFilterExample = new JLabel("e.g. 'Sea Zone'");		
+		
+		GridBagLayout gbl_panel_1 = new GridBagLayout();
+		gbl_panel_1.rowHeights = new int[] { 32, 32, 32, 32, 32, 32, 32, 0 };
+		gbl_panel_1.rowWeights = new double[] { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, Double.MIN_VALUE };
+		m_stepActionPanel.setLayout(gbl_panel_1);
+		
+		//Map Name
+		GridBagConstraints gbc_lMapName = new GridBagConstraints();
+		gbc_lMapName.insets = new Insets(0, 0, 5, 5);

```

Comment:
gbc_lMapName is the variable of the GridBagConstraints object that is responsible for variable lMapName. It makes a lot of sense to have some helping naming convention and they are not at all meaningless.
I will leave it as it is for the moment and you can find better names if you want. To me any other name can only make it more difficult to understand the dependencies between the many object.